src/__tests__/readArray.ts

```diff
@@ -99,45 +101,53 @@ describe('readArray', () => {
     expect(BeRes[1]).toBe(secondNumber);
   });
   it('float 32', () => {
-    const data = new Uint8Array([0, 2, 0, 1, 0, 0, 0, 1]);
-    //numbers taken from Buffer.readFloatBE and LE in Node.js
-    const le = 2.351132194607322e-38;
-    const le1 = 2.350988701644575e-38;
-    const be = 1.8367239361444675e-40;
-    const be1 = 1.401298464324817e-45;
+    const dataFromLE = new Uint8Array([1, 5, 3, 31, 3, 4, 40, 8]);
+    const dataFromBE = new Uint8Array([31, 3, 5, 1, 8, 40, 4, 3]);
+
+    const firstNumber = 2.774446815681537e-20;
+    const secondNumber = 5.056037679289265e-34;
 
     // little endian
-    let buffer = new IOBuffer(data);
+    let buffer = new IOBuffer(dataFromLE);
     const res = buffer.readArray(2, 'float32');
     expect(buffer.offset).toBe(8);
-    expect(res[0]).toBe(le);
-    expect(res[1]).toBe(le1);
+    expect(res[0]).toBe(firstNumber);
+    expect(res[1]).toBe(secondNumber);
 
     // big endian
+    buffer = new IOBuffer(dataFromBE);
```

Comment:
did you have to add this line? why?

Reply:
The previous version looked like this:
```
    const data = new Uint8Array([0, 2, 0, 1, 0, 0, 0, 1]);
    //numbers taken from Buffer.readFloatBE and LE in Node.js
    const le = 2.351132194607322e-38;
    const le1 = 2.350988701644575e-38;
    const be = 1.8367239361444675e-40;
    const be1 = 1.401298464324817e-45;
```
I did that because typing 2 arrays was more effort. The current version is symmetric in the sense that all the tests have the same logic, in words this is the current logic:
* Create u int 8 arrays of at least 2 "reads" of the type
* Revert each read, keeping the order 
* Test that the results are the same

I thought in this way you don't have to trust that the numbers are the ones I just happen to type, like those `be`, `be1`, though I still had to include `firstNumber` and `secondNumber`.

Reply:
Maybe even the best way would be to add `Buffer.from(uint8array).readFloatLE()` right into the test, so that the numbers are what Node outputs. I don't mind to do it.